tests/cephfs/snapshot_clone/cross_platform_snaps.py

```diff
@@ -0,0 +1,154 @@
+            log.info(
+                f"This test requires minimum 1 client nodes.This has only {len(clients)} clients"
+            )
+            return 1
+        default_fs = "cephfs"
+        mounting_dir = "".join(
+            random.choice(string.ascii_lowercase + string.digits)
+            for _ in list(range(10))
+        )
+        client1 = clients[0]
+        fs_details = fs_util.get_fs_info(client1)
+        if not fs_details:
+            fs_util.create_fs(client1, "cephfs")
+        subvolumegroup_list = [
+            {"vol_name": default_fs, "group_name": "subvol_cross_platform_snapshot_1"},
+        ]
+        for subvolumegroup in subvolumegroup_list:
+            fs_util.create_subvolumegroup(client1, **subvolumegroup)
+        subvolume = {
+            "vol_name": default_fs,
+            "subvol_name": "subvol_cross_platform_snapshot",
+            "group_name": "subvol_cross_platform_snapshot_1",
+            "size": "5368706371",
+        }
+        fs_util.create_subvolume(client1, **subvolume)
+        log.info("Get the path of sub volume")
+        subvol_path, rc = client1.exec_command(
+            sudo=True,
+            cmd=f"ceph fs subvolume getpath {default_fs} subvol_cross_platform_snapshot"
+            f" subvol_cross_platform_snapshot_1",
+        )
+        subvol_path = subvol_path.read().decode().strip()
+        fuse_mounting_dir_1 = f"/mnt/cephfs_fuse{mounting_dir}_1/"
+        fs_util.fuse_mount(
+            [client1],
+            fuse_mounting_dir_1,
+            extra_params=f" -r {subvol_path}",
+        )
+        fs_util.create_file_data(
+            client1, fuse_mounting_dir_1, 3, "snap1", "data_from_fuse_mount "
+        )
+        fuse_files_checksum = fs_util.get_files_and_checksum(
+            client1, fuse_mounting_dir_1
+        )
+        fuse_snapshot = {
+            "vol_name": default_fs,
+            "subvol_name": "subvol_cross_platform_snapshot",
+            "snap_name": "snap_1",
+            "group_name": "subvol_cross_platform_snapshot_1",
+        }
+        fs_util.create_snapshot(client1, **fuse_snapshot)
+        kernel_mounting_dir_1 = f"/mnt/cephfs_kernel{mounting_dir}_1/"
+        mon_node_ips = fs_util.get_mon_node_ips()
+        fs_util.kernel_mount(
+            [clients[0]],
+            kernel_mounting_dir_1,
+            ",".join(mon_node_ips),
+            sub_dir=f"{subvol_path}",
+        )
+        fs_util.create_file_data(
+            client1, kernel_mounting_dir_1, 3, "snap1", "data_from_kernel_mount "
+        )
+        kernel_snapshot = {
+            "vol_name": default_fs,
+            "subvol_name": "subvol_cross_platform_snapshot",
+            "snap_name": "snap_2",
+            "group_name": "subvol_cross_platform_snapshot_1",
+        }
+        fs_util.create_snapshot(client1, **kernel_snapshot)
+        kernel_files_checksum = fs_util.get_files_and_checksum(
+            client1, kernel_mounting_dir_1
+        )
```

Comment:
reboot &  remount steps missing?

Reply:
This was a documentation issue.
This test case does not require reboot and remount.
Updating the documentation